spring-cloud-huawei-config/src/main/java/com/huaweicloud/config/model/KVBody.java

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-package com.huaweicloud.config.kie;
+package com.huaweicloud.config.model;
```

Comment:
This model is only used for kie. So I think you do not give a properpy direrctory structure to hold different implementations.

Reply:
I think model is better than kie , it's hard to call a higher better name